src/test/java/ca/corefacility/bioinformatics/irida/model/UserTest.java

```diff
@@ -151,6 +151,16 @@ public void testInvalidEmail() {
 		assertEquals(b.getString("user.email.invalid"), constraintViolations.iterator().next().getMessage());
 	}
 
+	@Test
+	public void testNullUserType() {
+		User u = new User();
+		u.setUserType(null);
```

Comment:
Would it be better to just raise an Exception in `setUserType` if null was passed? Instead of having the null constraint, since this should never happen.